test/distributed/test_c10d_nccl.py

```diff
@@ -2685,7 +2685,11 @@ def assert_fut_success(fut):
             work = process_group.allreduce(torch.rand(10).cuda(self.rank))
             work.wait()
             result = work.get_future_result().wait()
-            self.assertEqual(WorkResult(result), WorkResult.COMM_ERROR)
+            self.assertEqual(WorkResult(result), WorkResult.TIMEOUT)
```

Comment:
nit: why do we need a type conversion here?

Reply:
There seems to be a type limitation of ivalue/future, we could not directly pass an enum to it, and we need to convert it an int.